testing-unit-common/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.bumble.appyx.testing.unit.common" />
```

Comment:
Can you please update the build.gradle namespace to 'com.bumble.appyx.testing.unit.common'?